docs/guide/catalog/set_up_products.md

```diff
@@ -0,0 +1,53 @@
+# Set up product for purchasing
+
+To enable purchasing from the catalog, the following configuration is required:
+
+- at least [one region and one currency for the shop](#region-and-currency)
+- [VAT rates for the product type](#vat-rates)
+- at least [one price for the product](#product-price)
+- [availability with positive or infinite stock for the product](#product-availability)
```

Comment:
How about adding direct links to Product price and Product availability. Currently there is one more click to get to the point, maybe we could reduce it to only one :)